src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/IVendorRegistrationFormService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.LegalEntities
+{
+    public interface IVendorRegistrationFormService
+    {
+        Task RefreshVendorRegistrationFormStatuses(DateTime fromDateTime, DateTime toDateTime);
```

Comment:
Can this be re-named to RefreshStatuses ?  It's on the IVendorRegistrationFormService so we know it applies to that.